arch/x86/real_mode.S

```diff
@@ -139,6 +139,15 @@ END_FUNC(_prot_to_real)
 
 .align 16
 .Lfrom_long_mode:
+    /* Use protected mode data selector */
+    mov   $__KERN_DS32, %eax
+    mov   %eax, %ds
+
```

Comment:
Should we set `%ss` as well, in case some future code tries to make use of the stack in this block?

Reply:
done